### ISSUE TITLE
nodeJS: More Detailed Explanation in 'Routes and Controllers' Lesson

### ISSUE THREAD

**Issue by thebranflakes** [TheOdinProject/curriculum#28061]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

Hi,

I wanted to provide a suggestion regarding the [Routes and Controllers lesson](https://www.theodinproject.com/lessons/nodejs-express-103-routes-and-controllers). I believe it would be beneficial for this page to be as in-depth and thoroughly explained as the previous lesson on [CRUD and MVC](https://www.theodinproject.com/lessons/nodejs-express-102-crud-and-mvc).

Although some of the material might have been mentioned in earlier lessons, re-explaining these concepts in detail would be extremely helpful for beginners. This extra clarity ensures a solid understanding as they progress through the tutorial.

Thank you.

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/nodejs-express-103-routes-and-controllers

**Comment by MaoShizhong** [comment]
Thanks for opening this issue @thebranflakes.
Improving these lessons like you're proposing is **a lot** of work, but I 125% agree with you...and am glad to let you know that we're already in the process of improving these lessons as part of a wider [Node course revamp](https://github.com/orgs/TheOdinProject/projects/38/views/4) :)

We haven't finished the revamp yet, nor have individual lessons been released yet, because the changes change more than just single lessons. So we need to finish at least this milestone before we can release anything.

**Comment by MaoShizhong** [comment]
Closing as this will be directly addressed by the revamped lessons
